### ISSUE TITLE
[Content styleguide] Content doesn’t align with VA.gov voice and tone content principles. (05.04.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#52236]
#### VFS team name
Sitewide Content

#### VFS product name
Authenticated 22-1990e Transfer of Entitlement (TOE) Form Updates

#### Point of Contact/Reviewers
Cheryl Evans - @CherylEvans - Content

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Content doesn’t align with VA.gov voice and tone content principles.

### Issue Details
What if the service member hasn’t requested a transfer of benefits yet? > “The service member needs to request a Transfer of Education Benefits (TEB) through milConnect first.” This sentence is somewhat unclear and hard to follow.

### VA.gov Experience Standard
[Category Number 05, Issue Number 04](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Platform Recommendation
Recommend rewriting: “The service member must first request a Transfer of Education Benefits (TEB) through milConnect.”

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by bethpottsVADEPO** [comment]
@CherylEvans @shiragoodman This sentence is grammatically correct and works in the context of the page and for the audience who would be reading this information. No edit needed. Closing ticket. 

@katherine-fung for awareness
